index.html

```diff
@@ -27,6 +27,11 @@
       shape-rendering: crispEdges;
     }
 
+    .axis .minor line {
+      stroke: #777;
+      stroke-dasharray: 2,2;
+    }
```

Comment:
Very nice. Just in case you haven't seem the explanation of the 2,2:
 https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray